src/net/mod.rs

```diff
@@ -2091,6 +2102,27 @@ pub trait Requestable: std::fmt::Display {
     fn make_request_type(&self, peer_host: PeerHost) -> HttpRequestType;
 }
 
+// TODO: DRY up from PoxSyncWatchdog
```

Comment:
It's fine if you want to address this in this PR :)

Reply:
I think that might not be straightforward given the structure of the codebase. Let me know if it is straightforward though.